static/docs/commands-reference/status.md

```diff
@@ -7,8 +7,8 @@ cache and remote cache.
 ## Synopsis
 
 ```usage
-usage: dvc status [-h] [-v] [-j JOBS] [--show-checksums] [-q] [-c]
-                  [-r REMOTE] [-a] [-T] [-d]
+usage: dvc status [-h] [-v] [-j JOBS] [--show-checksums]
+                  [-q] [-c] [-r REMOTE] [-a] [-T] [-d]
                   [targets [targets ...]]
```

Comment:
Hi. Why was this changed? In fact it can all fit in 2 lines:
```console
usage: dvc status [-h] [-v] [-j JOBS] [--show-checksums] [-q] [-c]
                  [-r REMOTE] [-a] [-T] [-d] [targets [targets ...]]
```

Reply:
p.s. I updated this in 5709d16 which is a change in my current PR, #464 so you can ignore this one but will probably cause a conflict before you can merge... Or feel free to also apply the same change in advance to avoid such Git conflict.